### DISCUSSION TITLE
Transfer joshbode/mise-sops

### DISCUSSION THREAD

**Discussion by joshbode** [mise-plugins/.github#10] [Transfer Request]
Hi! I would like discuss transferring [mise-sops](https://github.com/joshbode/mise-sops/) to `mise-plugins`, please.

This plugin was forked from [asdf-sops](https://github.com/feniix/asdf-sops/) and I added a `mise`-specific feature to allow automatic loading and export of variables from SOPS-encrypted files, optionally filtering which variables are exported.

I've been using it for a few weeks now and it's been working great for me :)

It potentially provides one solution to being able to manage secrets in a project (see jdx/mise#1359)

**Comment by jdx** [comment]
sounds good to me, go ahead and transfer it to me and I'll put it into the org

**Reply by joshbode** [reply]
thanks!

**Reply by joshbode** [reply]
Can I please have permission to contribute to the repo?

**Reply by jdx** [reply]
no, as mentioned in the README only myself and the advisory panel have that permission

**Reply by joshbode** [reply]
no worries - I've made a PR to update the links/instructions to point to the `mise-plugins` org
if you're going to be managing that yourself, that's fine with me :)
